src/interfaces/package-lock.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
That's weird -- how did this file get created? I don't see package-lock files for any other component in this repo.

Either all shared components should have a package-lock file, or none should (I think).

Reply:
ahh, perhaps during the hoist. Will clean these up

Reply:
Done 👍